src/js/select2/i18n/id.js

```diff
@@ -15,7 +15,7 @@ define(function () {
       return 'Masukkan ' + remainingChars + ' huruf lagi';
     },
     loadingMore: function () {
-      return 'Mengambil data…';
+      return 'Mengambil data ...';
```

Comment:
```suggestion
      return 'Mengambil data…';
```
The ellipsis change needs to be reverted.